website/docs/r/sql_database.html.markdown

```diff
@@ -48,3 +48,12 @@ In addition to the arguments listed above, the following computed attributes are
 exported:
 
 * `self_link` - The URI of the created resource.
+
```

Comment:
Thanks for the review @rileykarson. I've added all your suggestions however note that I now conflict as someone has already revised the documentation to make roughly the same change. Do you wish me to rebase?

```$ make testacc TESTARGS='-run=TestAccGoogleSqlDatabase_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccGoogleSqlDatabase_ -timeout 120m
?       github.com/terraform-providers/terraform-provider-google        [no test files]
=== RUN   TestAccGoogleSqlDatabase_importBasic
--- PASS: TestAccGoogleSqlDatabase_importBasic (67.64s)
=== RUN   TestAccGoogleSqlDatabase_basic
--- PASS: TestAccGoogleSqlDatabase_basic (66.62s)
PASS
ok      github.com/terraform-providers/terraform-provider-google/google 134.264s
```